compiler/AST/symbol.cpp

```diff
@@ -1453,7 +1466,7 @@ GenRet TypeSymbol::codegen() {
       // code generated, so code generate it now. This can get called
       // when adding types partway through code generation.
       codegenDef();
-      // codegenMetadata(); TODO -- enable TBAA generation in the future.
+       codegenMetadata(); //TODO -- enable TBAA generation in the future.
```

Comment:
Does this PR still work if you comment out codegenMetadata once again? I think we should be generating TBAA metadata but it was commented out because I havn't been sure it works correctly. Is this change one you inherited, or are you convinced codegenMetadata is correct?